avocado/plugins/nlist.py

```diff
@@ -70,7 +71,12 @@ def configure(self, parser):
     def run(self, config):
         references = config.get('nlist.references')
         verbose = config.get('nlist.verbose')
-        resolutions = resolver.resolve(references)
+        hint_filepath = '.avocado.hint'
+        if os.path.exists(hint_filepath):
+            hint = HintParser(hint_filepath)
+            resolutions = resolver.resolve(references, hint)
+        else:
+            resolutions = resolver.resolve(references)
```

Comment:
I find useful to maintain the code, when possible, using single calls to a method. In your case, as `resolutions` is always created calling `resolver.resolve()`, what do you think about a code like:

```
hint =  None
hint_filepath = '.avocado.hint'
if os.path.exists(hint_filepath):
    hint = HintParser(hint_filepath)
resolutions = resolver.resolve(references, hint)
```

It is just a cosmetic suggestion, won't change the logic, so feel free to ignore it.

Reply:
It is just a style that I follow when I have only 2 calls and also to avoid the `hint = None` line.

But, in this case, I don't see any problem with following your suggestion.